miniml/compiler/compile.scm

```diff
@@ -1594,6 +1594,19 @@
   (compile-expr compenv-empty 0
     (lower-expr env istail expr)))
 
+(define (substituable? expr)
+  (match expr
+   (((or 'LVar 'LGlobal 'LConst) . _)
+    #t)
+   (((or 'LBlock
+        'LGetfield 'Setfield
+        'LApply 'LTailApply
+        'LIf 'LChain 'LSwitch
+        'LReraise 'LCatch
+        'LLet 'LLetfun 'LLetrecfun) . _)
```

Comment:
It seems like `'LLetexits` and `LExit` are missing here.

Reply:
this is now fixed, thanks